platform/ui/src/components/Button/Button.mdx

```diff
@@ -5,9 +5,7 @@ route: components/button
 ---
 
 import { Playground, Props } from 'docz';
-import Button from './';
-import IconButton from '../IconButton';
-import Icon from './../Icon';
+import { Button, IconButton, Icon } from '@ohif/ui';
```

Comment:
Hmmmmmmmmmmm.... I think this is okay.
It feels weird to me, but if you and @rodrigolabs are in agreement then I'm cool with it.
Just make sure you don't push anything to a top level export when it shouldn't be to satisfy this.

Reply:
Yes, that's only for the exposed components. "Local" components should never be exposed.